docs/LV_client/utilities/mono_parse_depth_data.md

```diff
@@ -0,0 +1,32 @@
+# mono_parse_depth_data.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_parse_depth_data.png"/>
+</p>
+
+### Description
+
+Parse the raw data from the Depth Camera sensor and output the  normalized data.
+
+For technical support contact us at **support@monodrive.io** 
+
+### Inputs
+
+- **Distance (meters):**  Adjust the maximum distance an object can be seen
+ 
+
+- **Data:**  Raw binary data from the sensor
+ 
+
+- **error in (Error Cluster):** Accepts error information wired from previously called VIs. This information can be used to decide if any functionality should be bypassed in the event of errors from other VIs. 
+
+### Outputs
+
+- **Raw data:**  Data parsed into an array of double.
```

Comment:
does this make sense : "Data parsed into a double array"?

Reply:
An array of doubles in LV is similar to a list of fractional numbers in python i.e. [1.2, 3.0, 2.5] 
Changed it to a "Data parsed into a 1D array of type double."